testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5173,6 +5173,31 @@ void testBitGetFunc(SqlOperatorFixture f, String functionName) {
     f.checkString("to_base64(x'61')", "YQ==", "VARCHAR NOT NULL");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6812">[CALCITE-6812]
+   * Add base64 function (enabled in Hive library)</a>. */
+  @Test void testBase64() {
+    final SqlOperatorFixture f = fixture().withLibrary(SqlLibrary.HIVE);
+    f.setFor(SqlLibraryOperators.BASE64);
+    f.checkString("base64(cast('a' as binary))", "YQ==", "VARCHAR NOT NULL");
+    f.checkString("base64('')", "", "VARCHAR NOT NULL");
+    f.checkNull("base64(null)");
+    f.checkString("base64('This is a test String.')",
+        "VGhpcyBpcyBhIHRlc3QgU3RyaW5nLg==",
+        "VARCHAR NOT NULL");
+  }
+
+  @Test void testUnBase64() {
+    final SqlOperatorFixture f = fixture().withLibrary(SqlLibrary.HIVE);
+    f.setFor(SqlLibraryOperators.UN_BASE64);
+    f.checkString("unbase64('VGhpcyBpcyBhIHRlc3QgU3RyaW5nLg==')",
+              "This is a test String.",
+              "VARCHAR");
+    f.checkString("unbase64('YQ==')", "a", "VARCHAR");
+    f.checkString("unbase64('')", "", "VARCHAR");
+    f.checkNull("unbase64(null)");
```

Comment:
Can you add a test to check `IllegalArgumentException` with null result for unbase64.

Reply:
OK, would do it.